public/app/plugins/panel/canvas/CanvasContextMenu.tsx

```diff
@@ -77,9 +77,29 @@ export const CanvasContextMenu = ({ scene }: Props) => {
       />
     );
 
+    const editElementMenuItem = () => {
+      const onClickEditElementMenuItem = () => {
```

Comment:
minor... can this be inside the if?

that way the function is not created (and not used the 99%) of the time it is not selected

Reply:
🤔 oh wait -- it is already inside a `selectedElements.length >= 1`